pkgs/os-specific/linux/batman-adv/alfred.nix

```diff
@@ -15,7 +15,7 @@ stdenv.mkDerivation rec {
   buildInputs = [ gpsd libcap libnl ];
 
   preBuild = ''
-    makeFlags="PREFIX=$out PKG_CONFIG=${pkg-config}/bin/${pkg-config.targetPrefix}pkg-config"
+    makeFlags="PREFIX=$out"
   '';
```

Comment:
```suggestion
  makeFlags = [ "PREFIX=${placeholder "out"}" ];
```

Reply:
makeFlags is string not a list. I am not sure about why editorconfig-checker is failing.